src/App.tsx

```diff
@@ -15,37 +16,91 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum Way {
+  'Unknown' = 0,
+  'Info' = 'is-info',
+  'Success' = 'is-success',
+}
+
+function isValidWay(value: Way): value is Way {
+  return Object.values(Way).includes(value);
+}
```

Comment:
The `isValidWay` function is intended to check if a value is a valid member of the `Way` enum. However, since `Way` includes both string and numeric values, the check `Object.values(Way).includes(value)` may not work as expected. Consider explicitly checking for the known enum values.